openapi/atlas-api-transformed.yaml

```diff
@@ -5306,8 +5315,6 @@ components:
             compatibility for deprecated provider types and/or instance sizes.
           readOnly: true
           type: string
-      required:
```

Comment:
Discriminator field should be required. Lack of presence of requirement on some subfields usually means missconfiguration